docs/docsite/rst/dev_guide/migrating_roles.rst

```diff
@@ -0,0 +1,411 @@
+
+.. _migrating_roles:
+
+*************************************************
+Migrating Roles to Roles in Collections on Galaxy
+*************************************************
+
+You can migrate an existing standalone role into a collection-based role that you can host on Galaxy. This offers you more flexibility and resources to have your role within a collection. With Ansible collections on Galaxy, you can use a single collection as a delivery mechanism for many roles, releasing them as a single cohesive unit of re-usable automation. Inside a collection, custom plugins that used to be shipped in the role's :file:`library/`` directory can now be shared across all roles within a
+collection without duplicating them in each role's :file:`library/`` directory.
+
+You also need to migrate roles to collections if you want them as part of certified Ansible content.
+
+.. note::
+
+	If you want to import your collection to Galaxy, you need a `Galaxy namespace <https://galaxy.ansible.com/docs/contributing/namespaces.html>`_.
```

Comment:
I might move this to a "Prerequisites" section right above the migration instructions.